updater/templates/base/README.md

```diff
@@ -23,20 +23,20 @@ The quick start hasn't been written yet (contributions are welcome!). The quick
 
 {{displayName}} documentation is stored in a few places:
 
-1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/purescript-{{packageName}}).
+1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/{{repo}}).
```

Comment:
I don't know if this bit is actually correct; I think it might _have_ to be `purescript-{{packageName}}` to be valid. Do you know, @hdgarrood? If so I can continue hardcoding that for the time being.